.travis.yml

```diff
@@ -29,7 +30,8 @@ before_install:
 
 install:
   - python setup.py -q develop
-
+  # temp build fix
+  - pip install django==1.11.23
```

Comment:
I don't understand this line

Reply:
To avoid travis error (in that PR: https://github.com/openwisp/django-loci/pull/21) I had to use older version of ubuntu (trusty), this ubuntu distro has deprecated `sqlite` packages and they weren't working with `Django 2.*` versions therefore I overread installed via requirement `django` version in travis to not mess up requirements

Reply:
can you try removing the lines 33 and 34 to see if the build works?
The line `pip install django==1.11.23` is not acceptable because it binds the buid to an old version of django.

Reply:
In my comment above I meant `Django 2.1.*` or higher instead of `Django 2.*` , my bad

I had to change version of ubuntu to `trusty` because in other way (if version was newer) `travis` built would throw that error: https://github.com/openwisp/django-loci/pull/21#issuecomment-521583444 to avoid this I added `dist: trusty` in `travis` (the same thing is done in `openwisp-controller`, in `controller` I already have added issue for adding support for newer version of `ubuntu` in `travis`, should I add the same in other repos where `trusty` is used (`django-loci` obviously too) )

, but there is bad side of that namely in `Django 2.1` `django` dropped support for sqlite version used by `trusty` (and BTW it is not that easy to update the version of sqlite) therefore I had to install `Django` < `2.1` because in other way `travis will throw error`

Reply:
and the error which is occuring now in `travis` is throwing even when I undo all changes of the commit